### PR TITLE
Add endian support

### PR DESCRIPTION
**Pull request by lwandrebeck**

### CODE REVIEW

File: src/variable.rs [hunk 1]
```diff
@@ -56,6 +67,44 @@ impl Variable {
...
+				self.ncid,
+				self.varid,
+				&mut e,
+			))?;
+		}
+		match e {
+			NC_ENDIAN_NATIVE => Ok(Endianness::Native),
+			NC_ENDIAN_LITTLE => Ok(Endianness::Little),
+			NC_ENDIAN_BIG => Ok(Endianness::Big),
+			_ => Err(e.into())
```
Comment (magnusuMET) [inline src/variable.rs]: Either `NC_EVARMETA` or create a string with an error description (there is an &str into error implementation)

File: src/variable.rs [hunk 2]
```diff
@@ -56,6 +67,44 @@ impl Variable {
...
+		}
+    }
+    /// Set endianness of the variable. Must be set before inserting data
+    ///
+    /// `endian` can take a value 0..=2 with 0 being NC_ENDIAN_NATIVE,
+    /// 1 NC_ENDIAN_LITTLE, 2 NC_ENDIAN_BIG
```
Comment (magnusuMET) [inline src/variable.rs]: Outdated

File: src/variable.rs (hunk 1 above)
Comment (lwandrebeck) [reply src/variable.rs]: I’m afraid I don’t understand what you want.

File: src/variable.rs (hunk 2 above)
Comment (lwandrebeck) [reply src/variable.rs]: Should be better with latest commit.

Comment (magnusuMET) [reply src/variable.rs]: Looking good!

File: src/variable.rs (hunk 1 above)
Comment (magnusuMET) [reply src/variable.rs]: The value returned when querying the endianness is not an error type, and can't be converted by this `from` implmentation. You need to change this to either return `NC_EVARMETA.into()`, create a new error type for endianess, or return a string message.

**Comment by magnusuMET** [comment]
fixes #28

File: src/variable.rs (hunk 1 above)
Comment (lwandrebeck) [reply src/variable.rs]: Let’s go for NC_EVARMETA then.

**Comment by magnusuMET** [comment]
Thanks!

**Pull request merged**